### PR TITLE
Add create & getAll cases endpoints

### PR DESCRIPTION
**Pull request by ervuks**
Implements `create` and `getAll` endpoints for TestCase.

### CODE REVIEW

File: lib/test_case.dart [hunk 1]
```diff
@@ -53,60 +54,148 @@ class TestCase {
...
+  static Future<TestCase> create(
+    int sectionId, {
+    required String title,
+    int? templateId,
+    int? typeId,
+    int? priorityId,
+    String? estimate,
+    int? milestoneId,
+    String? refs,
+    Map<String, dynamic>? customValues,
```
Comment (Josiassejod1) [inline lib/test_case.dart:107]: alphabetize

File: lib/test_case.dart [hunk 2]
```diff
@@ -53,60 +54,148 @@ class TestCase {
...
       priorityId: json['priority_id'],
       refs: json['refs'],
       sectionId: json['section_id'],
       suiteId: json['suite_id'],
+      templateId: json['template_id'],
       title: json['title'],
       typeId: json['type_id'],
       updatedBy: json['updated_by'],
       updatedOn: updatedOn,
+      customAttributes: customAttributes['custom_attributes'],
```
Comment (Josiassejod1) [inline lib/test_case.dart:94]: alpha

File: lib/test_case.dart [hunk 3]
```diff
@@ -24,22 +25,22 @@ class TestCase {
...
     this.priorityId,
     this.refs,
     this.sectionId,
     this.suiteId,
+    this.templateId,
     this.title,
     this.typeId,
     this.updatedBy,
     this.updatedOn,
+    this.customAttributes,
```
Comment (tshedor) [inline lib/test_case.dart:43]: Alphabetized

File: test/test_case_test.dart [hunk 4]
```diff
@@ -5,10 +5,22 @@ import 'data/sample_test_case.dart';
 
 void main() {
   group('TestSection', () {
```
[the remark this answers is not in the document]
Comment (ervuks) [reply test/test_case_test.dart:7]: Good catch, yeah.

**Review by Josiassejod1 — APPROVED** [review]

**Pull request merged**